src/util/Util.js

```diff
@@ -33,23 +33,24 @@ class Util extends null {
     const out = {};
 
     for (let [prop, newProp] of Object.entries(props)) {
-      if (!newProp) continue;
-      newProp = newProp === true ? prop : newProp;
-
-      const element = obj[prop];
-      const elemIsObj = isObject(element);
-      const valueOf = elemIsObj && typeof element.valueOf === 'function' ? element.valueOf() : null;
-
-      // If it's a Collection, make the array of keys
-      if (element instanceof Collection) out[newProp] = Array.from(element.keys());
-      // If the valueOf is a Collection, use its array of keys
-      else if (valueOf instanceof Collection) out[newProp] = Array.from(valueOf.keys());
-      // If it's an array, flatten each element
-      else if (Array.isArray(element)) out[newProp] = element.map(e => Util.flatten(e));
-      // If it's an object with a primitive `valueOf`, use that value
-      else if (typeof valueOf !== 'object') out[newProp] = valueOf;
-      // If it's a primitive
-      else if (!elemIsObj) out[newProp] = element;
+      if (newProp) {
+        newProp = newProp === true ? prop : newProp;
+
+        const element = obj[prop];
+        const elemIsObj = isObject(element);
+        const valueOf = elemIsObj && typeof element.valueOf === 'function' ? element.valueOf() : null;
+
+        // If it's a Collection, make the variable named out to be the array of keys
+        if (element instanceof Collection) out[newProp] = Array.from(element.keys());
+        // If valueOf is a Collection, use its array of keys
+        else if (valueOf instanceof Collection) out[newProp] = Array.from(valueOf.keys());
+        // If it's an array, flatten each element
+        else if (Array.isArray(element)) out[newProp] = element.map(e => Util.flatten(e));
+        // If it's an object with a primitive `valueOf`, use that value
+        else if (typeof valueOf !== 'object') out[newProp] = valueOf;
+        // If it's a primitive, set out to be that primitive value
+        else if (!elemIsObj) out[newProp] = element;
+      }
```

Comment:
Please revert this, it was fine as it was, there's no real readability improvement here.

While you may think that sticking to basic syntax would increase the readability, the increase of curly braces can lead to the opposite effect, which is why that alongside to reduce the indentation in our code, we chose to use `continue` here.